rust/switchboard-v2/Cargo.toml

```diff
@@ -22,10 +22,10 @@ cpi = ["no-entrypoint"]
 devnet = []
 
 [dependencies]
-anchor-lang = "0.25.0"
-anchor-spl = "0.25.0"
-rust_decimal = { version = "1.18.0" }
-solana-program = "~1.10.29"
+anchor-lang = { git = "https://github.com/coral-xyz/anchor.git" }
+anchor-spl = { git = "https://github.com/coral-xyz/anchor.git" }
+rust_decimal = "1.26.0"
```

Comment:
this rust decimal change isn't necessary

Reply:
lgtm